### ISSUE TITLE
Support for `LayerNormalization` from ONNX opset 17

### ISSUE THREAD

**Issue by pwuertz** [onnx/tensorflow-onnx#2183]
When converting a `tensorflow.keras.layers.LayerNormalization` layer to ONNX, `tf2onnx` currently decomposes layer normalizations into rather complex subgraphs with batch norms and more basic building blocks. Inference engines (like TensorRT in the following example) are hardly able to deduce the original layer norm op from the graph and will have to follow the instructions to the letter:
![image](https://user-images.githubusercontent.com/1819283/241226042-408e0524-1100-48f2-a898-ce615ec9bdf7.png)

Since ONNX version 17 however, `tensorflow.keras.layers.LayerNormalization` layers are directly convertible to ONNX [LayerNormalization](https://github.com/onnx/onnx/blob/main/docs/Operators.md#layernormalization) operators. This of course produces a much simpler and expressive ONNX graph and leaves more room for the inference engine for possible optimizations (ONNX-to-TensorRT example again):
![image](https://github.com/onnx/tensorflow-onnx/assets/1819283/a6026d65-ce42-4106-aa1e-3c20b62c692e)

The current version of `tf2onnx` doesn't seem to produce those new `LayerNormalization` operators yet, at least not with `tf2onnx.convert.from_keras(model, opset=17)`.

**Comment by DayDayupupupup** [comment]
Same issue. Is there any plan to support rewrite LayerNormalization（opset 17）when use tf‘s savedmodel format？

**Comment by charvey2718** [comment]
Same issue for me.

**Comment by ShuaiShao93** [comment]
I have a tentative fix in https://github.com/onnx/tensorflow-onnx/pull/2250. Feel free to take it over

**Comment by YouSenRong** [comment]
Same issue for me.
I am converting the frozen_graph (TF 1.15) to ONNX for employing TensorRT to do inference in FP16 precision.
However, I get the warning as:
"[W] [TRT] Running layernorm after self-attention in FP16 may cause overflow. Exporting the model to the latest available ONNX opset (later than opset 17) to use the INormalizationLayer, or forcing layernorm layers to run in FP32 precision can help with preserving accuracy.".
Converting the frozen_graph to ONNX with Opset 17 does not help either. Finally I found that it due form tf2onnx.
If there is any progress, please let me know.
